Geofencing/Services/Managers/PlaceManager.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  PlaceManager.swift
+//  Geofencing
+//
+//  Created by Krešimir Baković on 25/03/2020.
+//  Copyright © 2020 Codable Studio. All rights reserved.
+//
+
+import Foundation
+
+class PlaceManager {
+    
+    static var shared = PlaceManager()
+    
+    var allPlaces = [Place]()
+}
```

Comment:
dodaj private init kako bi ovo stvarno bia singleton (https://medium.com/@nimjea/singleton-class-in-swift-17eef2d01d88)